compiler/resolution/functionResolution.cpp

```diff
@@ -4124,6 +4124,26 @@ void printResolutionErrorAmbiguous(CallInfo&                  info,
   USR_STOP();
 }
 
+static bool isMethodPreResolve(CallExpr* call) {
+  bool result = false;
+
+  if (call->numActuals() >= 2)
+    if (SymExpr* se = toSymExpr(call->get(1)))
+      result = se->symbol() == gMethodToken;
+
+  return result;
+}
+
+static bool isInitEqualsPreResolve(CallExpr* call) {
+  bool result = false;
+
+  if (isMethodPreResolve(call))
+    if (call->isNamedAstr(astrInit) || call->isNamedAstr(astrInitEquals))
```

Comment:
It's not clear to me how checking for `astrInit` would answer the `isInitEquals` predicate of this routine?

Reply:
Good catch, thank you!